.github/workflows/ci.yaml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+name: ci
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['unit-3.6', 'unit-3.7', 'unit-pypy3']
```

Comment:
```suggestion
        python: ['unit-3.6', 'unit-3.7']
```
Session `'unit-pypy3'` was removed from [noxfile.py](https://github.com/googleapis/python-ndb/blob/master/noxfile.py) in https://github.com/googleapis/python-ndb/pull/286